govcd/lbserverpool_test.go

```diff
@@ -0,0 +1,105 @@
+// +build lb lbServerPool functional ALL
+
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+// Test_LBServerPool tests CRUD methods for load balancer server pool.
+// The following things are tested:
+// Creation of load balancer server pool
+// Read load balancer server pool by both ID and Name (server pool name must be unique in single edge gateway)
+// Update - change a single field and compare that configuration and result objects are deeply equal
+// Update - try and fail to update without mandatory field
+// Delete
+func (vcd *TestVCD) Test_LBServerPool(check *C) {
```

Comment:
Maybe worth mentioning prerequisites of this test (EG).

Reply:
Added a note about it.